boms/src/main/resources/generated-project-template.xml

```diff
@@ -24,11 +24,26 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>BOM_GROUPID</groupId>
+        <artifactId>bom-certified</artifactId>
```

Comment:
`bom-certified` only exists in product, while this template seems universal?

Reply:
Good point. @michalszynkiewicz `additionalBom` is not needed in community.

Reply:
@mkouba @Ladicek wdyt about adding another template, for prod, and using it only in prod profile?

Reply:
Works for me :-)

Reply:
I don't think we need to maintain a special template for prod in community repo, i.e. `generated-project-template.xml ` in community repo does not have to be equal to `generated-project-template.xml ` in prod repo.

Reply:
That works too, but generally the less difference between upstream and downstream repos, the better. And everything related to `bom-certified` is already in the upstream repo.